tests/test_saml2_client.py

```diff
@@ -70,6 +82,13 @@ def test_artifact_resolve_content_type_settings(self):
 
 @pytest.mark.usefixtures("eherkenning_config_defaults", "temp_private_root")
 class EHerkenningClientTests(TestCase):
+    def setUp(self):
```

Comment:
always make sure to call the `super().setUp()` in case mixins get added in the future :)